re_data_ui/src/components/Sidebar.tsx

```diff
@@ -1,47 +1,50 @@
                 transform md:relative md:translate-x-0 transition duration-200 ease-in-out  md:flex
                  md:flex-col md:justify-between max-h-screen overflow-y-auto" data-dev-hint="sidebar; px-0 for frameless; px-2 for visually inset the navigation">
             <div className="flex flex-col space-y-6"
                  data-dev-hint="optional div for having an extra footer navigation">
                 <a href="/" className="text-white flex items-center space-x-2 px-4" title="Your App is cool">
-                    <svg xmlns="http://www.w3.org/2000/svg" className="h-6 w-6 flex-0" fill="none"
-                         viewBox="0 0 24 24" stroke="currentColor">
-                        <path strokeLinecap="round" strokeLinejoin="round" strokeWidth="2"
-                              d="M5 3v4M3 5h4M6 17v4m-2-2h4m5-16l2.286 6.857L21 12l-5.714 2.143L13 21l-2.286-6.857L5 12l5.714-2.143L13 3z"/>
-                    </svg>
-                    <span className="text-2xl font-extrabold whitespace-nowrap truncate">re data</span>
+                    <span className="text-2xl font-extrabold whitespace-nowrap truncate">
+                        <img alt="re_data" className="w-14" src="/redata.svg" />
+                    </span>
                 </a>
 
                 <nav data-dev-hint="main navigation">
                     <Link to='alerts'
                           className="flex items-center space-x-2 py-2 px-4 transition duration-200 hover:bg-gray-700 hover:text-white">
-                        <ImNotification/>
+                        <ImNotification size="1.25em" />
                         <span>Alerts</span>
                     </Link>
 
                     <Link to='graph'
                        className="flex items-center space-x-2 py-2 px-4 transition duration-200 hover:bg-gray-700 hover:text-white">
-                        <BiNetworkChart/>
+                        <BiNetworkChart size="1.25em" />
                         <span>Graph</span>
                     </Link>
                 </nav>
             </div>
 
             <nav data-dev-hint="second-main-navigation or footer navigation">
-                <a href="/" className="block py-2 px-4 transition duration-200 hover:bg-gray-700 hover:text-white">
-                    Slack
+                <a href="https://www.getre.io/slack" className="flex items-center space-x-2 py-2 px-4 transition duration-200 hover:bg-gray-700 hover:text-white">
+                    <BsSlack size="1.25em" />
+                    <div>Slack</div>
                 </a>
-                <a href="/" className="block py-2 px-4 transition duration-200 hover:bg-gray-700 hover:text-white">
-                    Docs
+                <a href="https://re-data.github.io/re-data/" className="flex items-center space-x-2 py-2 px-4 transition duration-200 hover:bg-gray-700 hover:text-white">
+                    <SiReadthedocs size="1.25em" />
+                    <span>Docs</span>
```

Comment:
I'm thinking this should open in a new tab using this attribute.
```
target="_blank"
```

Reply:
done